### PR TITLE
v1.0.0-alpha.0

### PR DESCRIPTION
**Pull request by jessicamcinchak**
Queuing up a pre-release !

After this merges, will run `npm publish --tag next` rather than typical `npm publish` per thread here: https://gist.github.com/schmich/d04acc4b02b45e489f329cfdf3280a3f?permalink_comment_id=4483624#gistcomment-4483624

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- The `areaChange` event dispatched in `drawMode` has been deprecated and rolled into existing `geojsonChange` event. If your `drawType="Polygon"`, you'll now simply find an `area` property on the dispatched geojson feature ([#466-discussion](https://github.com/theopensystemslab/map/pull/466#discussion_r1703872391))
+- Similarly, the `featuresAreaChange` event dispatched by `clickFeatures` has been deprecated and rolled into `featuresGeojsonChange` event ([#479](https://github.com/theopensystemslab/map/pull/479))
+- `areaUnit` prop has been deprecated and you'll find the calculated area in _both_ `squareMetres` and `hectares` by default now in `geojsonChange` event data above ([#479](https://github.com/theopensystemslab/map/pull/479))
+
+We think the above deprecations will mean simpler, _more_ flexible configurations and subscriptions for end-users, but if you were relying on any of the deprecated props, cannot achieve feature parity with the alternatives, or find a regression we've overlooked, please open an [Issue](https://github.com/theopensystemslab/map/issues)!
```
Comment (DafyddLlyr) [inline CHANGELOG.md:25]: Tone is great, nice to highlight this and encourage issues to be opened 👌

**Review by DafyddLlyr — APPROVED** [review]
I know we're awaiting a few more merges, but all here looks good to me!

**Comment by jessicamcinchak** [comment]
`npm publish --tag next` worked exactly as expected :white_check_mark: 

Screenshot from 2024-08-26 13-06-51

**Comment by DafyddLlyr** [comment]
Nice! 🎉

**Pull request merged**